iOS/Schema/Sources/Action/Types/Navigate/Navigate.swift

```diff
@@ -14,24 +14,37 @@
  * limitations under the License.
  */
 
+/// Handles screens navigations actions of the application.
 public enum Navigate: RawAction {
-    
+    /// Opens up an available browser on the device and navigates to a passed URL as String.
     case openExternalURL(String)
+    /// Opens up a route to execute an action declared on defined deeplink in the application.
```

Comment:
I think that using the term "execute an action" was a kind of misleading here.
I suggest something more like:
```suggestion
    /// Opens a screen that is defined all locally in your app (does not depend on Beagle) which will be retrieved using `DeeplinkScreenManager`.
```